frontend/apps/reader/modules/readergesture.lua

```diff
@@ -239,6 +240,14 @@ function ReaderGesture:initGesture()
     G_reader_settings:saveSetting(self.ges_mode, gesture_manager)
 end
 
+function ReaderGesture:onReSetupGesture()
```

Comment:
Not the biggest fan of ReInitGesture either, but at least it's internally consistent. ;-)
```suggestion
function ReaderGesture:onReInitGesture()
```